Hermes/appscale/hermes/stats/subscribers/profile.py

```diff
@@ -1,19 +1,70 @@
+import csv
+from os import path, makedirs
+
+from appscale.hermes.stats import converter
+from appscale.hermes.stats.producers import node_stats
 from appscale.hermes.stats.pubsub_base import StatsSubscriber
+from appscale.hermes.stats.constants import PROFILE_LOG_DIR
 
 
 class ClusterNodesProfileLog(StatsSubscriber):
 
+  def __init__(self, include_lists=None):
+    """ Initializes profile log for cluster node stats.
+    Renders header according to include_lists in advance and
+    creates base directory for node stats profile log.
+
+    Args:
+      include_lists: an instance of IncludeLists describing which fields
+          of node stats should be written to CSV log.
+    """
+    self._header = converter.get_stats_header(
+      node_stats.NodeStatsSnapshot, include_lists)
+    self._include_lists = include_lists
+    self._directory = path.join(PROFILE_LOG_DIR, 'node')
+    if not path.exists(self._directory):
```

Comment:
EAFP. I think https://github.com/AppScale/appscale/blob/master/AdminServer/appscale/admin/utils.py#L98-L104 is the common convention.